tests/foreman/api/test_role.py

```diff
@@ -115,8 +156,16 @@ def test_positive_create_role_with_taxonomies(self):
 
```

Comment:
Pls remove `:caseautomation: notautomated`

Reply:
Yeah, I have taken care of that